### PR TITLE
Adding support for buildslave tags in Katana

### PR DESCRIPTION
**Pull request by BarryUnity**

### CODE REVIEW

File: www/prod/script/main.js [hunk 1]
```diff
@@ -51027,6 +51033,22 @@ define('rtBuildSlaves',['require','jquery','realtimePages','helpers','project/da
 
                 },
                 rtTable.cell.slaveHealth(5)
+
+                // TODO: this should remain uncommented until moving Katana to Dublin
```
Comment (campos-ddc) [inline www/prod/script/main.js:51037]: Should remain uncommented but is commented?

**Review by campos-ddc — COMMENTED** [review]
Why do we need tags for builder?

File: www/prod/script/main.js (hunk 1 above)
Comment (BarryUnity) [reply www/prod/script/main.js:51037]: Whoops. Yes, it's meant to say commented, or dummied out.

**Comment by BarryUnity** [comment]
Daniel's request. Presumably, it's so that he doesn't need to go to Yumi to see what a builder is tagged as being able to do. Since it's a small change, I have no objection.

**Review by kkateq — APPROVED** [review]

**Comment by mariangemarcano** [comment]
@BarryUnity could you fix things that are commented and also make it so if there are not tags we don't display and empty element and formatted like this Tags: windows, IntelTablet, xBOX (coma separated, right now looks like this windows,IntelTablet,xBOX)

**Comment by mariangemarcano** [comment]
Closing this PR for now, to finish up this task it should be in sync with configuration changes to add this support.

**Pull request closed without merging**